src/main/xar-resources/data/urlrewrite/urlrewrite.xml

```diff
@@ -10,175 +8,169 @@
         </listitem>
-      </varlistentry>
-      <varlistentry>
-        <term>exist:resource</term>
         <listitem>
-          <para>The section of the URI after the last
-            <literal>/</literal>, usually pointing to a resource.
-          </para>
-          <para>For instance:
-            <literal>example.xml</literal>.</para>
+          <para>The <code>XQueryUrlRewrite</code> servlet looks for an XQuery script to interpret the rest of the URI (see <xref linkend="controller-mappings"/>).  By convention this is called <code>controller.xq</code> (or in older applications <code>controller.xql</code>).  The default configuration will work with a script saved with this name in the base collection of an application.</para>
         </listitem>
-      </varlistentry>
-      <varlistentry>
-        <term>
-          <code>exist:controller</code>
-        </term>
         <listitem>
-          <para>The part of the URI leading to the current controller script.</para>
-          <para>For example: if the request path is
-            <literal>/xquery/test.xql</literal>
-            and the controller is in the
-            <literal>xquery</literal>
-            collection,
-            <literal>$exist:controller</literal>
-            will contain
-            <literal>/xquery</literal>.</para>
+          <para><code>controller.xq</code> examines the URI using pre-defined <xref linkend="variables"/>, and produces a fragment in the <xref linkend="controller-xml"/>.</para>
         </listitem>
-      </varlistentry>
-      <varlistentry>
-        <term>
-          <code>exist:prefix</code>
-        </term>
         <listitem>
-          <para>If the current controller hierarchy is mapped to a certain path prefix,
-            <literal>$exist:prefix</literal>
-            returns that prefix.</para>
-          <para>
-            For example: the default configuration maps the path
-            <literal>/tools</literal>
-            to a collection in the database (see below). In this case,
-            <literal>$exist:prefix</literal>
-            would contain
-            <literal>/tools</literal>.</para>
+          <para><code>XQueryURLRewrite</code> uses the controller XML fragment as an instruction on what to do next.  This instruction may be as simple as forwarding to a resource on (or off) the server, or it may be as complex as a pipeline using the <emphasis>Model-View-Controller</emphasis> pattern (see <xref linkend="mvc-pipelines"/>) and other servlets such as eXist's <xref linkend="xq-servlet"/> or <xref linkend="xslt-servlet"/>.</para>
         </listitem>
-      </varlistentry>
-      <varlistentry>
-        <term>
-          <code>exist:root</code>
-        </term>
+      </orderedlist>
+    <sect2 xml:id="eg1">
+      <title>Example I: A Simple Implementation</title>
```

Comment:
The next example is "Example 2", so perhaps best to stick with Arabic numerals -> "Example 1".